pulsar-broker/src/main/java/org/apache/pulsar/PulsarClusterMetadataSetup.java

```diff
@@ -63,10 +66,18 @@
  */
 public class PulsarClusterMetadataSetup {
 
+    private static final int DEFAULT_BUNDLE_NUMBER = 16;
+
     private static class Arguments {
         @Parameter(names = { "-c", "--cluster" }, description = "Cluster name", required = true)
         private String cluster;
 
+        @Parameter(names = {"-bc", "--broker-conf"},
+                description = "Configuration file for Broker to get the default bundle numbers "
+                        + "for public/default namespace")
+        private String brokerConfigFile =
+                Paths.get("").toAbsolutePath().normalize().toString() + "/conf/broker.conf";
```

Comment:
Do we need this value? This will break the `DEFAULT_BUNDLE_NUMBER`.

Reply:
This is an optional configuration. Users can set this if they want to set the bundle number for the `public/default` namespace.

Reply:
Oh, I see, but I suggest  using the `private String brokerConfigFile;`.

Reply:
Fixed